feat: Added Letter Spacing option

mpa12 opened the pull request:
## Summary

Added Letter Spacing option

resolves #251 

## Type of change

- [x] New feature (added a non-breaking change which adds functionality)

## How Has This Been Tested?

- [x] Ran tests with `composer test`
- [x] Added or updated test cases to test new features

DenverCoder1 commented on src/demo/index.php:
```diff
@@ -70,6 +70,9 @@ function gtag() {
                 <label for="size">Font size</label>
                 <input class="param" type="number" id="size" name="size" alt="Font size" placeholder="20" value="20">
 
+                <label for="letterSpacing">Font letter spacing</label>
```
A tooltip could help here to explain how to enter a letter spacing value

```suggestion
                <div class="label-group">
                    <label for="letterSpacing">Font letter spacing</label>
                    <span class="icon tooltip" title="Enter a CSS letter-spacing value with units (eg. '5px', '0.5em'). Defaults to 'normal'.">
                        <svg stroke="currentColor" fill="currentColor" stroke-width="0" viewBox="0 0 24 24" height="1em" width="1em" xmlns="http://www.w3.org/2000/svg">
                            <path d="M12 6C9.831 6 8.066 7.765 8.066 9.934h2C10.066 8.867 10.934 8 12 8s1.934.867 1.934 1.934c0 .598-.481 1.032-1.216 1.626-.255.207-.496.404-.691.599C11.029 13.156 11 14.215 11 14.333V15h2l-.001-.633c.001-.016.033-.386.441-.793.15-.15.339-.3.535-.458.779-.631 1.958-1.584 1.958-3.182C15.934 7.765 14.169 6 12 6zM11 16H13V18H11z"></path>
                            <path d="M12,2C6.486,2,2,6.486,2,12s4.486,10,10,10s10-4.486,10-10S17.514,2,12,2z M12,20c-4.411,0-8-3.589-8-8s3.589-8,8-8 s8,3.589,8,8S16.411,20,12,20z"></path>
                        </svg>
                    </span>
                </div>
```

DenverCoder1 reviewed (commented):
Looks great! Thanks for the contribution :tada:

mpa12 commented:
Some error in "Format with Prettier"

```
Run EndBug/add-and-commit@v9
  with:
    message: style: Formatted code with Prettier
    default_author: github_actions
    add: .
    cwd: .
    fetch: --tags --force
    pathspec_error_handling: ignore
    push: true
    github_token: ***
Running in /home/runner/work/readme-typing-svg/readme-typing-svg
Add input parsed as single string, running 1 git add command.
> Using 'github-actions <41898[2](https://github.com/DenverCoder1/readme-typing-svg/actions/runs/9760791461/job/26940859740?pr=302#step:7:2)82+github-actions[bot]@users.noreply.github.com>' as author.
> Using "style: Formatted code with Prettier" as commit message.
Internal logs
  > Staging files...
  > Adding files...
  > No files to remove.
  > Checking for uncommitted changes in the git working tree...
  > Found 5 changed files.
  > Fetching repo...
  {
    raw: '',
    remote: null,
    branches: [],
    tags: [],
    updated: [],
    deleted: []
  }
  > Not pulling from repo.
  > Creating commit...
  {
    author: null,
    branch: 'main',
    commit: '28e0c4[3](https://github.com/DenverCoder1/readme-typing-svg/actions/runs/9760791461/job/26940859740?pr=302#step:7:3)f41a473cd8e214d42c2eda2a4d76b9652',
    root: false,
    summary: { changes: 5, insertions: 10, deletions: 28 }
  }
  > No tag info provided.
  > Pushing commit to repo...
  Error: Error: Pushing to https://github.com/mpa12/readme-typing-svg
  remote: Permission to mpa12/readme-typing-svg.git denied to github-actions[bot].
  fatal: unable to access 'https://github.com/mpa12/readme-typing-svg/': The requested URL returned error: [4](https://github.com/DenverCoder1/readme-typing-svg/actions/runs/9760791461/job/26940859740?pr=302#step:7:4)03
  
Outputs
  committed: true
  commit_long_sha: 28e0c43f41a473cd8e214d42c2eda2a4d76b96[5](https://github.com/DenverCoder1/readme-typing-svg/actions/runs/9760791461/job/26940859740?pr=302#step:7:5)2
  commit_sha: 28e0c43
  pushed: false
  tagged: false
  tag_pushed: false
Error: Error: Pushing to https://github.com/mpa12/readme-typing-svg
remote: Permission to mpa[12](https://github.com/DenverCoder1/readme-typing-svg/actions/runs/9760791461/job/26940859740?pr=302#step:7:13)/readme-typing-svg.git denied to github-actions[bot].
fatal: unable to access 'https://github.com/mpa12/readme-typing-svg/': The requested URL returned error: 403
```

DenverCoder1 commented:
Thanks!
